parser-typechecker/src/Unison/CommandLine/Main.hs

```diff
@@ -74,23 +74,23 @@ getUserInput
   -> Path.Absolute
   -> [String]
   -> m Input
-getUserInput patterns codebase branch currentPath numberedArgs =
-  Line.runInputT settings go
+getUserInput patterns codebase branch currentPath numberedArgs = Line.runInputT
+  settings
+  go
  where
   go = do
-    line <- Line.getInputLine $
-      P.toANSI 80 ((P.green . P.shown) currentPath <> fromString prompt)
+    line <- Line.getInputLine
+      $ P.toANSI 80 ((P.green . P.shown) currentPath <> fromString prompt)
     case line of
       Nothing -> pure QuitI
-      Just l ->
-        case words l of
-          [] -> go
-          ws ->
-            case parseInput patterns . (>>= expandNumber numberedArgs) $ ws  of
-              Left msg -> do
-                liftIO $ putPrettyLn msg
-                go
-              Right i -> pure i
+      Just l  -> case words l of
+        [] -> go
+        ws ->
+          case parseInput patterns . (>>= expandNumber numberedArgs) $ ws of
+            Left msg -> do
+              liftIO $ putPrettyLn msg
+              go
+            Right i -> pure i
```

Comment:
Formatting changes only. There was a change here but it got changed back at some point.